pkg/apis/duck/v1alpha1/resource_types_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2020 The Knative Authors
```

Comment:
check that the copyrights are Google LLC
here and every new filed that you added